### DISCUSSION TITLE
Git push

### DISCUSSION THREAD

**Discussion by jaydezi-code** [community/community#134361] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I’m trying to do a push request but for some reason when it asks for my password it’s either not letting me see it or it’s not typing it at all and I even reset my password to a fairly easy to remember password.

**Comment by anderson895** [comment]
Double-check that you're entering the correct password. Since you mentioned resetting it to an easy-to-remember one, make sure you're typing it correctly

**Reply by jaydezi-code** [reply]
Ok thanks for the help

**Comment by davevad93** [comment]
Hi @jaydezi-code ,

When typing the password in `git bash` it doesn't show any characters, this is the standard behavior, also the password depends on whether you are doing `git` operations with `HTTPS` or `SSH`. For `HTTPS` you have to create a `Personal Access Token` (with the `repo` scopes) and use it as password while with `SSH` the password is the one set when generating the ssh key.

Here's the documentation about generating a `PAT` and a `SSH` key:

- [Creating a personal access token (classic)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
- [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)

**Comment by cyndie-lauper** [comment]
hi @jaydezi-code, when u type your password in the terminal, it usually doesn't show anything to keep it secure. Just type your password and press Enter. If it’s still not working, double-check that you’ve copied the new password correctly :>

**Reply by jaydezi-code** [reply]
Ok thanks for the advice
